packages/front-end/hooks/useOrganizationMetricDefaults.ts

```diff
@@ -0,0 +1,35 @@
+import { MetricDefaults } from "back-end/types/organization";
+import { useEffect, useState } from "react";
+import useOrgSettings from "./useOrgSettings";
+
+const defaultMaxPercentChange = 0.5;
+const defaultMinPercentChange = 0.005;
+const defaultMinSampleSize = 150;
+
+const METRIC_DEFAULTS = {
+  minimumSampleSize: defaultMinSampleSize,
+  maxPercentageChange: defaultMaxPercentChange,
+  minPercentageChange: defaultMinPercentChange,
+};
+
+/**
+ * Metric defaults are stored at the organization settings level.
+ * If an organization has them set, use them, otherwise use the hardcoded defaults.
+ */
+export const useOrganizationMetricDefaults = (): MetricDefaults => {
+  const [metricDefaults, setMetricDefaults] = useState<MetricDefaults>(
+    METRIC_DEFAULTS
+  );
+
+  const orgSettings = useOrgSettings();
+
+  useEffect(() => {
+    const updatedDefaults: MetricDefaults = {
+      ...METRIC_DEFAULTS,
+      ...(orgSettings?.metricDefaults || {}),
+    };
+    setMetricDefaults(updatedDefaults);
+  }, [orgSettings?.metricDefaults]);
```

Comment:
Thoughts on replacing `useState` + `useEffect` with a single `useMemo` call?  I'm fine either way

```tsx
export const useOrganizationMetricDefaults = (): MetricDefaults => {
  const orgSettings = useOrgSettings();

  return useMemo<MetricDefaults>(
    () => ({
      ...METRIC_DEFAULTS,
      ...(orgSettings?.metricDefaults || {}),
    }),
    [orgSettings?.metricDefaults]
  );
};
```

Reply:
That would totally work. I can make the change.

Reply:
I think this might actually fix the `form.reset` issue too. You were using `useState` which started with the default value. Then `useEffect` would fire and cause a re-render with the updated value. That's why `useForm` was only picking up the default value since it caches on the initial render. With `useMemo`, it returns the correct value the first time, so it should work (at least in theory)

Reply:
Yes you're right, `useMemo` does seem to fix that issue. Not sure why I didn't think to use it this time, I often do but didn't think of it. Thanks!